modules/aruco/include/opencv2/aruco/dictionary.hpp

```diff
@@ -84,14 +84,14 @@ class CV_EXPORTS_W Dictionary {
     /**
      * @see generateCustomDictionary
      */
-    CV_WRAP_AS(create) static Ptr<Dictionary> create(int nMarkers, int markerSize);
+    CV_WRAP_AS(create) static Ptr<Dictionary> create(int nMarkers, int markerSize, int randomSeed=~0);
```

Comment:
Please use "cv::uint64" type for seed. "int" is not a good case for cross-platform support.

Why is `~0` seed value used here?
Simple "0" should work too:
```
inline RNG::RNG(uint64 _state) { state = _state ? _state : 0xffffffff; }
```

Reply:
> Why is ~0 seed value used here?

i was too lazy to type `0xffffffff;` , but sure, 0 is the better choice.

> why is it int, not uint64 ?
(setRNGSeed() wants it so.)
https://github.com/opencv/opencv/blob/0366c1b0932141d6f1a3d49d996d8aca22330a7e/modules/core/include/opencv2/core.hpp#L2271

hmm, there is support for `int64` in python/java, but not for `uint64`. would a `size_t` do ?

Reply:
`size_t` is not good solution for cross-platform too (it can have 32/64-bits).
setRNGSeed() uses `int` but RNG() uses `uint64`. This is not consistent behavior in OpenCV for now.
Perhaps you are right and the "int" type is required for bindings.

Reply:
so, int64 (truncates sign) or int ? (truncates 32bits)

Reply:
Lets use "int" to avoid issues with bindings. 32-bit is enough for RNG generator for now (and it can be easily extended to 64-bit without huge breaking of existed code)

Reply:
ok. will do that.